io.gdevelop.ide.yml

```diff
@@ -0,0 +1,54 @@
+# This is the Flatpak manifest for GDevelop.
+app-id: io.gdevelop.ide
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=all
+  - --filesystem=home
+  - --filesystem=xdg-run/dconf
+  - --filesystem=~/.config/dconf:ro
+  - --filesystem=xdg-run/pipewire-0
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+  - --talk-name=ca.desrt.dconf
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.gtk.Notifications
+  - --env=XDG_CURRENT_DESKTOP=Unity
```

Comment:
????

Reply:
Sorry about these, I based the file on https://github.com/flathub/com.github.IsmaelMartinez.teams_for_linux/blob/master/com.github.IsmaelMartinez.teams_for_linux.yml in the hope that the set of permissions/requirements is large enough to run everything that a game can do in GDevelop.

Seems it was to remove the titlebar somehow in the app, I'll remove this.